src/index.test.ts

```diff
@@ -123,3 +123,68 @@ it('convert var declarations to let declarations', () => {
 
   expect(actual).toBe(expected);
 });
+
+test(`api`, () => {
```

Comment:
I'm not sure what you are testing here.

Reply:
So, there's an else case that is very hard to get to inside of the `./transform#api.tscodeshift` function, and this test covers that else case. Sorry, I should have clarified my test case after I got it working. (Basically the coverage was lowered by me adding the files I added, so by covering the else case, I was able to get coveralls to pass.)

Reply:
Specifically this: https://travis-ci.org/KnisterPeter/tscodeshift/builds/237974069#L217